x-pack/legacy/plugins/siem/public/components/charts/common.tsx

```diff
@@ -144,5 +144,25 @@ export const getTheme = () => {
   return mergeWithDefaultTheme(theme, defaultTheme);
 };
 
+export const chartDefaultSettings = {
+  rotation: chartDefaultRotation,
+  rendering: chartDefaultRendering,
+  animatedData: false,
+  showLegend: false,
+  showLegendDisplayValue: false,
+  debug: false,
+  theme: getTheme(),
+};
+
 const kibanaTimezone: string = chrome.getUiSettingsClient().get(DEFAULT_DATE_FORMAT_TZ);
 export const browserTimezone = kibanaTimezone === 'Browser' ? moment.tz.guess() : kibanaTimezone;
+
+export const getChartHeight = (customHeight?: number, autoSizerHeight?: number): string => {
+  const height = customHeight || autoSizerHeight;
+  return height ? `${height}px` : defaultChartHeight;
+};
```

Comment:
All this code is really clean and easy to read! Thank you. I like the explicit return types you added. I do the same thing and that is a good recommendation from TypeScript.

Optional, I would ensure for pure functions we have unit tests even if they seem overly simple such as this one. Appreciate all the tests you have added here for this PR.